packages/components/view-switcher/src/view-switcher.spec.js

```diff
@@ -131,6 +132,34 @@ describe('rendering', () => {
     screen.getByLabelText('Test Button 1').click();
     expect(handleClick).not.toHaveBeenCalled();
   });
+
+  it('should be controlled when selectedValue is passed', () => {
```

Comment:
In terms of naming I suggested `selectedValue` as a pairing to `defaultValue`. I tried implementing `value` and `initialValue` but see two downsides. One it requiring the component to be backwards compatible to avoid a breaking change plus the additional toil work. Two the naming on Group and Button becomes a bit confusing in terms of what `value` on a Group means vs. a Button.

Reply:
I think it makes sense yes, thanks for thinking about it.